src/scala/scripts/ScroogeWorker.scala

```diff
@@ -58,8 +58,13 @@ object ScroogeWorker extends Worker.Interface {
 
     val scrooge = new Compiler
 
-    if (additionalFlags.contains("--with-finagle")) {
-      scrooge.flags += WithFinagle
+    additionalFlags.foreach {
+      case "--with-finagle" => {
+        scrooge.flags += WithFinagle
+      }
+      case f if f.startsWith("--language=") => {
```

Comment:
I'm personally not a fan of silently ignoring flags, but can definitely ignore them if we prefer to keep semantics. Or add a better error message.

Reply:
Better error message sounds ok.
MatchError sounds like a bad compromise for me since our users will need to dig into our code to understand what happened.
AFK so don’t remember- can the flags be passed by our users or does the rule fully control them? If they can be passed then we need a test for this error case

Reply:
The rule fully controls them, there's no sort of "extra scrooge flags" argument.

Reply:
Added error message on match failed.